sabnzbd/config.py

```diff
@@ -786,7 +786,6 @@ def _read_config(path, try_backup=False):
 def save_config(force=False):
     """ Update Setup file with current option values """
     global CFG, database, modified
-    assert isinstance(CFG, configobj.ConfigObj)
```

Comment:
The isinstance() calls are there to help static analysers understand the code.
Removing them is a loss of information.
The alternative is:
if 0: assert isinstance(CFG, configobj.ConfigObj)
This should incur no performance penalty.